src/main/java/xyz/nucleoid/plasmid/game/config/GameConfig.java

```diff
@@ -117,12 +165,16 @@ public String translationKey() {
         }
 
         private static <T, A> Optional<A> tryDecode(Codec<A> codec, DynamicOps<T> ops, T input) {
-            return codec.decode(ops, input).result().map(Pair::getFirst);
+            try {
+                return codec.decode(ops, input).result().map(Pair::getFirst);
+            } catch (Exception e) {
+                return Optional.empty();
```

Comment:
I'm not too great with codecs so possibly this solution was wrong. It throws an error for games with missing fields I wanted to make optional (name/description/etc)

Reply:
`DataResult.result()` will return `Optional.empty` in the case that the result is an error (when the field is missing) - so it should be safe in that regard given we're already handling the optionals here. Though is it possible for `TextParser.parseSafe` to throw an error? In that case the exception handling would be needed- but probably best to handle that within the codec itself since a codec should never throw an exception on its own.